src/svg/bounce.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M19.707 10.707L12 18.414l-5.5-5.5-4.5-4.5L3.414 7l3.793 3.793L12 15.586l6.293-6.293L16 7h6v6l-2.293-2.293z"/></svg>
```

Comment:
💅🏻  [nitpick]
Missing a new line at the end of the file

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M19.707 10.707L12 18.414l-5.5-5.5-4.5-4.5L3.414 7l3.793 3.793L12 15.586l6.293-6.293L16 7h6v6l-2.293-2.293z"/></svg>

```

Reply:
Custom icons are not editable manually, when commiting they're [processed by SVGO](https://github.com/onfido/castor-icons/blob/main/.lintstagedrc.json#L5) (which is optimizing by also removing the new line).